locales/locales.php

```diff
@@ -2596,6 +2633,17 @@ public function __construct() {
 		$yor->google_code = 'yo';
 		$yor->facebook_locale = 'yo_NG';
 
+		$zgh = new GP_Locale();
+		$zgh->english_name = 'Tamazight';
+		$zgh->native_name = 'ⵜⴰⵎⴰⵣⵉⵖⵜ';
+		$zgh->lang_code_iso_639_2 = 'zgh';
+		$zgh->lang_code_iso_639_3 = 'zgh';
+		$zgh->country_code = 'MA';
+		$zgh->wp_locale = 'zgh';
+		$zgh->slug = 'zgh';
+		$zgh->nplurals = 2;
+		$zgh->plural_expression = 'n==0 && n==1 && n>=11 || n<=99 ? 0 : 1';
```

Comment:
```suggestion
		$zgh->plural_expression = 'n >= 2 && (n < 11 || n > 99)';
```

This will match the [CLDR data for Tamazight](https://php-gettext.github.io/Languages/).